docs/CHANGELOG.md

```diff
@@ -0,0 +1,49 @@
+Spectacle Docs
```

Comment:
Should the docs maintain their own, separate CHANGELOG? Or should we keep the one that was in the repo there for archival purposes, and then centralize all changes to one CHANGELOG at the root?

Reply:
I don't think the docs need a separate changelog going forward.